winui/Kanban/swimlane.md

```diff
@@ -0,0 +1,220 @@
+---
+layout: post
+title: Swimlane support in WinUI Kanban control | Syncfusion
+description: Learn here all about the swimlane support and its customization in Syncfusion WinUI Kanban (SfKanban) control.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+
+# Swim lanes in WinUI Kanban Control
+
+The Swim lanes organize the Kanban board horizontally, allowing you to categorize cards by key attributes such as assignee, project, or task type.
+
+By default, swimlanes are categorized based on the `Assignee` property in the `KanbanModel` class. However, you can customize the categorization by mapping the `SwimlaneKey` property to the appropriate property name in `KanbanModel`.
+
+The following code example demonstrates how to group the underlying data collection using the `SwimlaneKey` property:
+
+{% tabs %}
+{% highlight XAML tabtitle="MainWindow.xaml" hl_lines="2" %}
+
+<kanban:SfKanban x:Name="kanban"
+                    ItemsSource="{Binding TaskDetails}"
+                    SwimlaneKey="IndicatorColorKey">
+    <kanban:SfKanban.DataContext>
+        <local:ViewModel/>
+    </kanban:SfKanban.DataContext>
+</kanban:SfKanban>
+
+{% endhighlight %}
+
+{% highlight C# tabtitle="MainWindow.xaml.cs" %}
+
+this.kanban.SwimlaneKey = "IndicatorColorKey";
```

Comment:
Add all properties, as used in XAML and highlight SwimlaneLey

Reply:
Added.